java/org/apache/tomcat/jni/Library.java

```diff
@@ -177,6 +177,9 @@ private Library(String libraryName)
     /* Is the O_NONBLOCK flag inherited from listening sockets?
      */
     public static boolean APR_O_NONBLOCK_INHERITED  = false;
+    /* Support for Unix Domain Sockets.
+     */
+    public static boolean APR_HAVE_UNIX           = false;
```

Comment:
I had the same on my mind, but this is the define from APR. It should be consistent. Look into the header files.

Reply:
Over at APR we call it APR_UNIX, it would be better to be consistent with this:

https://github.com/apache/apr/blob/trunk/include/apr_network_io.h#L168